data/sql/updates/pending_db_world/rev_1553712779275952300.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1553712779275952300');
+
+DELETE FROM conditions WHERE SourceTypeOrReferenceId=15 AND SourceGroup=1050;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`SourceId`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionTarget`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`,`NegativeCondition`,`ErrorType`,`ErrorTextId`,`ScriptName`,`Comment`) VALUES
+(15, 1050, 0, 0, 0, 2, 0, 9281, 1, 0, 0, 0, 0, '', 'Display option if player has red card'),
+(15, 1050, 1, 0, 0, 2, 0, 9281, 1, 0, 0, 0, 0, '', 'Display option if player has red card'),
+(15, 1050, 1, 0, 0, 2, 0, 9327, 1, 0, 0, 0, 0, '', 'Display option if player has delta card'),
+(15, 1050, 1, 0, 0, 25, 0, 3959, 0, 0, 1, 0, 0, '', 'Display option if player has no Discombobulator Ray spell');
+
```

Comment:
Note that you don't need the last line (number 9) in further PRs because the travis bot adds it by itself (unless I'm totally wrong). However a carriage return is always needed on the end of line 8 (on linux it's automatic)

Reply:
> Note that you don't need the last line (number 9) in further PRs because the travis bot adds it by itself (unless I'm totally wrong). However a carriage return is always needed on the end of line 8 (on linux it's automatic)

Okay :+1: